<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>나만의 투자봇</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Chapter-6/icon.ico</normaloff>../Chapter-6/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>141</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>기준봉</string>
    </property>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>30분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton60">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>60분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton240">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>240분</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1일</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중지</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRange">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>매도 구간</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRange">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTarget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTarget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>대상 코인</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSecret">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecret">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>API SECRET</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccess">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>API ACCESS</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditAccess">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
